packages/components/src/Tooltip/src/Tooltip.module.css

```diff
@@ -0,0 +1,67 @@
+.hop-Tooltip {
+    --hop-Tooltip-max-inline-size: 25rem;
+    --hop-Tooltip-slide-amount: 0.25rem; /* Tokens not available here */
```

Comment:
can't use it here, that's why i added the comment :P 
Basically, tooltip is portaled, which means it's no longer inside the HopperProvider, so we add it inside the Tooltip, bad sadly, we don't have it on .hop-Tooltip, only .hop-Tooltip__container, but I wanted the positioning on the root element, which is why I just hardcoded it here.

Reply:
If you wrapped the HopperProvider by the Tooltip, this wouldn't work neither?

Reply:
i'd really like to know if there is another possibility for this. 
it's really sad that we don't reference our tokens.

instead of putting it on the tooltip, or the tooltip container, is it possible to put it on the HopperProvider (that renders a div?)

Reply:
I'm not sure how that improves things?
The provider would work basically like the tooltip container so it could allow me to remove the extra div for tooltip container and use the provider. The issue here is that the transform needs to be on the tooltip, main div. Same thing for any widths or max-widths, it need to be on the tooltip. I wish that all the provider details were added to the html tag or body :( so then we wouldn't have to do this.

Reply:
i didn't want to remove an extra div, i wanted to avoid hardcoding tokens. 

Can't the transformation be on an inner div? like if we don't put anything on the tooltip, and everything in another div inside the tooltip?

Reply:
I mean it might work.. if position is relative. It's just not really the cleanest way. 
Plus I found out, I may have to do a similar thing with the width like we did for popover so width definitely needs to stay on .hop-Tooltip.
But I can try moving the transform if you really want.

Reply:
it somehow didn't work so we are going to keep it where it is for now.